test/org/sagebionetworks/bridge/dynamodb/DynamoStudyTest.java

```diff
@@ -79,6 +81,9 @@ public void studyFullySerializesForCaching() throws Exception {
         study = BridgeObjectMapper.get().readValue(json, DynamoStudy.class);
         assertNull(study.getStormpathHref());
         assertEquals("Study", node.get("type").asText());
+
+        // Deserialize back to a POJO and verify.
+        DynamoStudy deserStudy = BridgeObjectMapper.get().readValue(json, DynamoStudy.class);
```

Comment:
I figured it was okay for DynamoStudyTest to know about DynamoStudy. That said, for this specific line, we deserializing as a Study instead of a DynamoStudy works just fine, so I went ahead and changed it.

Also, I realized the test as written re-uses variables a lot, which means the last test for deserialization actually doesn't do anything useful. So I slightly refactored the test.